content/vi/docs/reference/glossary/container.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Container
+id: container
+date: 2019-29-11
+full_link: /docs/concepts/overview/what-is-kubernetes/#why-containers
+short_description: >
+  Là một image nhẹ và di động có khả năng thực thi chứa ứng dụng và những dependencies đi kèm.
```

Comment:
Should be: "Một image nhẹ, khả chuyển và có khả năng thực thi, chứa phần mềm và tất cả các dependencies của nó."